docs/RunningFirstBenchmark.md

```diff
@@ -0,0 +1,114 @@
+# Running Your First Benchmark
+
+This tutorial is aimed at people new to the library and shows you how to run benchmarks with the Google Benchmark library.
+
+# Install the prerequisites
```

Comment:
This should only be necessary if a package is not available. Ie, it might be worth having a section on getting the library first, with installation from source being one of the options.